.travis/build-pkgs

```diff
@@ -8,8 +8,9 @@ set -eu
 # changed/created in the specified revisions.
 changed_repos() {
 	local commit_ish="$1"
+	local files="$(find -type f -maxdepth 1 | sed 's:^\./::' | tr '\n' '|' | sed 's:|$::')"
 
-	git diff-tree --name-only "$commit_ish" | grep -v '^\..*' || echo ''
+	git diff-tree --name-only "$commit_ish" | grep -Ev -e '^\..*' -e $files || echo ''
```

Comment:
This is unnecessarily complicated. There’s a more straightforward and simpler way, see [here](https://github.com/alpinelinux/aports/pull/115/commits/960e0dad422116d9dbba27fa6bc08638e44f3526)